app/client/sagas/filterSaga.ts

```diff
@@ -50,9 +51,10 @@ function* watchLoadFilters() {
   yield takeEvery(LoadFilterActionTypes.LOAD_FILTERS_REQUEST, loadFilters);
 }
 
-function* createFilter({ payload }) {
+function* createFilter(action: CreateFilterAction) {
   try {
-    const result = yield call(createFilterAPI, payload);
+    //@ts-ignore
+    const result = yield call(createFilterAPI, action.payload);
```

Comment:
It seems the generic action creator isn't working as intended... @EDsCODE if you have a spare few minutes could you see if you can investigate this?

```
Error:(56, 55) TS2339: Property 'payload' does not exist on type 'ActionWithPayload<CreateFilterActionTypes.CREATE_FILTER_REQUEST, CreateFilterPayload> | Action<CreateFilterActionTypes.CREATE_FILTER_SUCCESS> | ActionWithError<...> | Action<...>'.
  Property 'payload' does not exist on type 'Action<CreateFilterActionTypes.CREATE_FILTER_SUCCESS>'.
```

Reply:
https://github.com/teamsempo/SempoBlockchain/pull/275/commits/9a8c3fefd0f9fc557dfdf90ec1fbeb9a5a33c5a7#r429002799